starknet_py/net/client.py

```diff
@@ -213,3 +217,24 @@ async def add_transaction(
         :return: Dictionary with `code`, `transaction_hash`
         """
         return await self._gateway.add_transaction(tx, token)
+
+    async def deploy(
+        self,
+        compiled_contract: Optional[str],
+        constructor_calldata: List[int],
+        salt: Optional[int] = None,
+    ) -> dict:
+        res = await self.add_transaction(
+            tx=Deploy(
+                contract_address_salt=ContractAddressSalt.get_random_value()
+                if salt is None
```

Comment:
maybe `salt or ContractAddressSalt.get_random_value()` would be more readable

Reply:
salt can be `0`